ACTUALLY removes whitelist from most of the donator items, other than the carrot stuff, because a certain user really wants to keep them to thereselves

thestubborn opened the pull request:
see #17 and #14 

current mood: 
![bunbunonthelaplap](https://user-images.githubusercontent.com/58394696/73799241-49207680-47ad-11ea-8697-111f71fa25b3.png)

thestubborn committed:
i will scream

death

thestubborn committed:
hope this time it works, i am 100% sure i whitelisted that damn cloak

bazinga

thestubborn commented:
oh and because i'm an idiot, the plushee thing is here as well....enjoy

thestubborn committed:
the tale of reese the reasonable

thestubborn commented:
right, if the green coat one ig, wearing, still has the reece thing, i'll fix that

improvedname commented on modular_citadel/code/modules/client/loadout/__donator.dm:
```diff
@@ -179,19 +176,16 @@
 	name = "Carrot Satchel"
```
Keep the whitelist

thestubborn replied:
check the lateist commit

improvedname commented on modular_citadel/code/modules/client/loadout/__donator.dm:
```diff
@@ -106,7 +106,7 @@
 	name = "Carrot plushie"
```
Im fine with this one

thestubborn committed:
i forgot the carrot satchel

Oh I know Jeremy Harrington well, too well sadly He lived in a richer area of New York at one point. I bumped into him in a sandwich shop. He was shooting the shit with the sandwich maker when I came up to give my order. I got a standard turkey on wheat with mayo and pepper jack. It was something very simple. All the while Jerma was laughing to himself and pointing at me occasionally mumbling, "Is this guy serious?"

I wasn't sure why I guess he was displeased about me ordering or what I ordered. After I got my sandwich I sat down and ate while he just leaved back into the windowed meat display and watched me. When I got up to throw the trash away he stopped me and offered his hand. I was shy because I knew who he was but he introduced himself anyways. When I shook his hand the other hand came out of his pocket holding a tazer which he used on my neck. When I fell to the ground tense I soiled myself. He laughed, "This guy." And started kicking me in the face quoting videos over and over, "DON'T SAY SWEARS, DON'T SAY SWEARS."

I woke up in the hospital next to a vase of broccoli labeled, "From Jerma."

Now every year he shows up to my families Thanksgiving dinner, uninvited, unannounced, and just eats, laughs to himself, and then leaves. Last year he murdered my mother. I fear what hurricane Jerma will bring this year...

thestubborn committed:
restricts the mime overalls to the mime, restricts a coat to the science team, and makes a scarf delisted

    STEP ONE: Secure the keys.
    STEP TWO: Ascend from darkness.
    STEP THREE: Rain fire.
    STEP FOUR: Unleash the horde.
    STEP FIVE: Skewer the winged beast.
    STEP SIX: Wield a fist of iron.
    STEP SEVEN: Raise Hell.
    STEP EIGHT: Freedom.
